Generated code is breaking `cargo test` because of its doc-test contains a moved import

### Describe the bug

`flutter_rust_bridge::frb_generated_boilerplate!` contains a doc test which is breaking since > 2.0.0-dev.18.


### Steps to reproduce

1. See https://github.com/CaviarChen/flutter_rust_bridge/commit/5c9460094ab197ad095974f7fd497d9a7dccdf54 All I did is add "lib" to crate-type so one could run `cargo test`.
2. run `cargo test`
3. you get:
```
running 1 test
error[E0412]: cannot find type `RustOpaque` in this scope
  --> src/frb_generated.rs:47:29
   |
7  | pub struct DebugWrapper(pub RustOpaque<Box<dyn Debug>>);
   |                             ^^^^^^^^^^ help: a struct with a similar name exists: `DartOpaque`
   |
  ::: ~/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-2.0.0-dev.21/src/dart_opaque/mod.rs:30:1
   |
30 | pub struct DartOpaque {
   | --------------------- similarly named struct `DartOpaque` defined here

error[E0412]: cannot find type `RustOpaque` in this scope
  --> src/frb_generated.rs:52:30
   |
12 | pub struct DebugWrapper2(pub RustOpaque<Box<dyn Debug + Send + Sync + UnwindSafe + RefUnwindSafe>>);
   |                              ^^^^^^^^^^ help: a struct with a similar name exists: `DartOpaque`
   |
  ::: ~/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-2.0.0-dev.21/src/dart_opaque/mod.rs:30:1
   |
30 | pub struct DartOpaque {
   |
```

### Logs

```shell
n/a
```


### Expected behavior

_No response_

### Generated binding code

_No response_

### OS

_No response_

### Version of `flutter_rust_bridge_codegen`

2.0.0-dev.21

### Flutter info

_No response_

### Version of `clang++`

_No response_

### Additional context

_No response_

--- a/frb_rust/src/rust_opaque/mod.rs
+++ b/frb_rust/src/rust_opaque/mod.rs
@@ -29,31 +29,6 @@ macro_rules! frb_generated_rust_opaque_def {
         pub type RustOpaqueMoi<T> = $crate::for_generated::RustOpaqueBase<T, MoiArc<T>>;
 
         /// A wrapper to support [arbitrary Rust types](https://cjycode.com/flutter_rust_bridge/guides/types/arbitrary).
-        ///
-        /// ## Naming the inner type
-        ///
-        /// When an `RustOpaque<T>` is transformed into a Dart type, T's string
-        /// representation undergoes some transformations to become a valid Dart type:
-        /// - Rust keywords (dyn, 'static, etc.) are automatically removed.
-        /// - ASCII alphanumerics are kept, all other characters are ignored.
-        ///
-        /// ## Trait objects
-        ///
-        /// Trait objects can be put behind opaque pointers. For example, this declaration can
-        /// be used across the FFI border:
-        ///
-        /// ```rust
-        /// use flutter_rust_bridge::*;
-        /// use std::fmt::Debug;
-        /// use std::panic::{UnwindSafe, RefUnwindSafe};
-        ///
-        /// pub struct DebugWrapper(pub RustOpaque<Box<dyn Debug>>);
-        ///
-        /// // creating a DebugWrapper using the opaque_dyn macro
-        /// let wrap = DebugWrapper(opaque_dyn!("foobar"));
-        /// // it's possible to name it directly
-        /// pub struct DebugWrapper2(pub RustOpaque<Box<dyn Debug + Send + Sync + UnwindSafe + RefUnwindSafe>>);
-        /// ```
         pub type RustOpaque<T> = $default_rust_opaque<T>;
     };
 }

--- a/website/docs/guides/types/arbitrary/rust-opaque.md
+++ b/website/docs/guides/types/arbitrary/rust-opaque.md
@@ -29,6 +29,27 @@ await api.functionThatUsesSomeOpaqueData(opaque);
 opaque.dispose();
 ```
 
+## Trait objects
+
+Trait objects can be put behind opaque pointers. For example, this declaration can
+be used across the FFI border:
+
+```rust
+pub struct DebugWrapper(pub RustOpaque<Box<dyn Debug>>);
+
+// creating a DebugWrapper using the opaque_dyn macro
+let wrap = DebugWrapper(opaque_dyn!("foobar"));
+// it's possible to name it directly
+pub struct DebugWrapper2(pub RustOpaque<Box<dyn Debug + Send + Sync + UnwindSafe + RefUnwindSafe>>);
+```
+
+## Naming the inner type
+
+When an `RustOpaque<T>` is transformed into a Dart type, T's string
+representation undergoes some transformations to become a valid Dart type:
+- Rust keywords (dyn, 'static, etc.) are automatically removed.
+- ASCII alphanumerics are kept, all other characters are ignored.
+
 ## Implementation details
 
 As for how it is implemented as well as the design towards safety,

